data/ecosystems/a/actions-marketplace-validations.toml

```diff
@@ -0,0 +1,6 @@
+title = "actions-marketplace-validations"
+sub_ecosystems = []
+github_organizations = ["https://github.com/actions-marketplace-validations"]
```

Comment:
As a critical example of what my main comment was referring to. This org has 18.6k repos, and since you'd be adding it as an org to ICP, all those repos would get linked to ICP, and not only the FleekHQ one, like it should.

Again, better to keep the repos in the ICP toml and crete ecosystems only when it makes sense. And only add orgs when you are sure all its repos should belong to that sub-eco.

Reply:
thank you for those insights! that helps a lot :) i think i will manually review the creation of new ecosystems as it seems like its something thats not easy to automate

Reply:
totally. In fact, we have been experimenting with ML to classify repos and ecosystems and it's not an easy task. We are in the process of semi-automating a part of it (eg cases where the classification is obvious)